src/Lesson1/Task4.js

```diff
@@ -0,0 +1,12 @@
+//4*. Написать цикл, который создаёт массив промисов, внутри каждого промиса происходит обращение к ресурсу (https://jsonplaceholder.typicode.com/users/number), 
+//где вместо number подставляется число от 1 до 10, в итоге должно получиться 10 промисов. Следует дождаться выполнения загрузки всеми промисами и далее вывести массив загруженных данных.
+
+const fetch = require('node-fetch');
+const promises = [];
+
+for (let i = 1; i<11;i++){
+    promises.push(fetch('https://jsonplaceholder.typicode.com/users/'+i).then(response=>response.json().then(result=>console.log(result))).catch(error=>console.log(error)));
+}
+
+Promise.all(promises);
```

Comment:
```suggestion
Promise.all(promises).then(console.log);
```
мы дожидаемся выполнения всех обещаний, и только потом запускаем выполнение console.log()